web/src/components/MemoEditor.tsx

```diff
@@ -90,6 +94,19 @@ const MemoEditor: React.FC = () => {
       editorRef.current?.insertText(" ".repeat(TAB_SPACE_WIDTH));
       return;
     }
+    if (event.ctrlKey || event.metaKey) {
+      switch (event.key) {
+        case "b":
+          handleInsertMark("**");
+          break;
+        case "i":
+          handleInsertMark("*");
+          break;
+        case "~":
+          if (event.shiftKey) handleInsertMark("`");
+          break;
+      }
+    }
```

Comment:
return here can make git diff look better when a new key event added
```suggestion
      return;
    }
```